libs/tangle-shared-ui/src/types/index.ts

```diff
@@ -119,3 +120,7 @@ export type BridgeTokenWithBalance = BridgeToken & {
 export type BridgeChainBalances = Partial<
   Record<PresetTypedChainId, BridgeTokenWithBalance[]>
 >;
+
+export type RestakeAssetId = `${bigint}` | EvmAddress;
+
+export type TangleAssetId = { Custom: u128 } | { Erc20: Address };
```

Comment:
So that we have a bit stronger types when creating asset IDs for transactions:
```suggestion
export type TangleAssetId = { Custom: u128 } | { Erc20: EvmAddress };
```